modules/games/definitions/outwit/flow.ts

```diff
@@ -0,0 +1,43 @@
+import { OutwitDefinition } from "./_types";
+
+const outwitFlow: OutwitDefinition["flow"] = {
+  endGame: {
+    winline: {
```

Comment:
The name of the win condition is actually surfaced in the app. When show a finished session, it will say something like "Player 1 won via `winline`". So it is semi-important that the name of the win condition matches what it actually tests.

```suggestion
    infiltration: {
```

Reply:
Yeah I saw the text when I "won" but wasn't sure if it ment anything or now 😅